testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveStorageFormats.java

```diff
@@ -554,7 +543,7 @@ public void testStructTimestampsFromHive(StorageFormat format)
         onPresto().executeQuery(format("DROP TABLE %s", tableName));
     }
 
-    @Test(dataProvider = "storageFormatsWithNanosecondPrecisionExceptParquet", groups = STORAGE_FORMATS)
+    @Test(dataProvider = "storageFormatsWithNanosecondPrecision", groups = STORAGE_FORMATS)
```

Comment:
Didn't you remove detailed timestamps tests from storage format group?

Reply:
Apparently, I removed the regular timestamp tests, but not the struct-timestamp tests. Oops.